File: 1768. Merge Strings Alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string merged = "";
        int minLength = min(word1.size(), word2.size());

        for(int i = 0; i<minLength; i++){
            merged += word1[i];
            merged += word2[i];
        }

        merged.append(word1.begin()+minLength,word1.end());
        merged.append(word2.begin()+minLength,word2.end());

        return merged;
    }
};

File: 1768. Merge Strings Alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merged = [] 

        minLength = min(len(word1),len(word2))

        for i in range(minLength):
            merged.append(word1[i])
            merged.append(word2[i])
        
        merged.append(word1[minLength:])
        merged.append(word2[minLength:])

        return ''.join(merged)
